src/AutoRest.CSharp/Common/Output/Expressions/Statements/DeclareFieldStatement.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using AutoRest.CSharp.Common.Output.Expressions.ValueExpressions;
+
+namespace AutoRest.CSharp.Common.Output.Expressions.Statements
+{
+    internal record DeclareFieldStatement(VariableReference Variable) : MethodBodyStatement;
```

Comment:
I did not find a place where we are using this, maybe we could remove it?
Also I think we already have a similar statement, we do not have to have a declaration statement specifically for fields.